Packs/Base/Scripts/DBotUpdateLogoURLPhishing/DBotUpdateLogoURLPhishing.py

```diff
@@ -0,0 +1,183 @@
+import demistomock as demisto
```

Comment:
I think this script share too much functionality with the other one.
Maybe we should you the original one with special arguments and this script will just do "executeCommand" with those special arguments. Let's talk about it

Reply:
The code seems to be similar but there are many small difference. I think it's more convenient to have 2 separate scripts rather than running one from the other one.

Reply:
It can be harder to maintain, because most of the code is similar and duplicated - so if you change something in the logic you have to change it on both scripts

Reply:
+ it's on Base, it means all customers will see this script and will try to understand what it does, might be confusing as it's kind of helper for another script